airbyte-webapp/src/views/Connection/ConnectionForm/components/SchemaChangeBackdrop.module.scss

```diff
@@ -16,7 +16,7 @@
     @extend %cover;
 
     position: absolute;
-    background: linear-gradient(180deg, rgba(colors.$grey-50, 0.5) 0%, colors.$white 92.54%);
+    background: linear-gradient(180deg, rgba(248, 248, 250, 50% /* grey-50 50% */) 0%, colors.$white 92.54%);
```

Comment:
Fun bug

Reply:
Yeah using `rgba()` with color variables will no longer work. My suggestion is to treat opaque variants of theme colors as their own variables (see e.g. the overlay background color [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/scss/_theme-light.scss#L91)) or, if they're really one-off colors, to define them inline like [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/packages/cloud/views/auth/components/PersonQuoteCover/PersonQuoteCover.module.scss#L26-L32). 

In this case I'm not sure which is more appropriate, probably best to define it inline like you did. But basically I would decouple it from `grey-50`, since it's a different color.